docs/io/visualization/generating_widgets.ipynb

```diff
@@ -5,16 +5,17 @@
    "metadata": {},
```

Comment:
<p>I guess we have to use the American spelling to be consistent...</p><p>visualise --&gt; visualize</p><p> </p><p>"The horizontal lines represent the energy levels. The thickness of each line shows the relative population of that energy level, with thicker lines being more populated."</p><p>"The arrows represent the line interactions between levels, with the arrow direction giving the direction of the transition. The thickness of each arrow also shows the number of packets that underwent the transition while the wavelength is given by the color."</p><p> </p><p>"In addition, you can also select between linear- and log-scaling for the y-axis (which represents the energy of each level) and the maximum number of levels to display, beginning from the lowest energy levels."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>